caravel/helpers.py

```diff
@@ -120,4 +126,76 @@ def _version_text(sep):
     from _version import __version__ as caravel_version
     from looper import __version__ as looper_version
     return "caravel version: {}".format(caravel_version) + sep + \
-           "looper version: {}".format(looper_version)
+           "looper version: {}\n".format(looper_version)
+
+
+def print_terminal_width(txt=None, char="-"):
+    """
+    Print a line composed of the chars and a text in the middle of the terminal
+
+    :param str txt: a string to display in the middle of the terminal
+    :param str char: a character that the box will be composed of
+    :return None
+    """
+    char = str(char)
+    assert len(char) == 1, "The length of the char parameter has to be equal 1, got '{}'".format(len(char))
+    spaced_txt = txt.center(len(txt)+2) if txt is not None else ""
+    print(char * int((terminal_width() / 2) - (len(spaced_txt) / 2)) + spaced_txt + char *
```

Comment:
I think this would read a little easier as something like:
```
fill_width = int(0.5 * (terminal_width() - len(spaced_text)))
filler = char * fill_width
print(filler + spaced_text + filler)
```